src/components/Tagline/Tagline.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+import styles from "./Tagline.module.scss";
+
+export function Tagline() {
+  return (
+    <main className={styles.main}>
```

Comment:
<main> is used to define the main content of a page. so if you use <main>, you would want to be wrapping the entire homepage basically. here we would just use h2 or h3